command/genesis/genesis.go

```diff
@@ -153,6 +154,14 @@ func (c *GenesisCommand) DefineFlags() {
 		},
 		FlagOptional: true,
 	}
+
+	c.FlagMap["max-validator-count"] = helper.FlagDescriptor{
+		Description: "Sets the flag indicating the maximum number of validators in the validator set for PoS",
```

Comment:
Please add what is the default value / or indicate that there is no upper bound if this flag isn't set 🙏